### PR TITLE
fix: 'confirm' modal returns when validation finished.

### PR DESCRIPTION
**Pull request by Nosafi**
'Confirm' modal returns when validation finished.

### CODE REVIEW

File: web/src/connectors/edit-job-connector/edit-job-connector.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
     showNoExtractionTab
 }) => {
     const getMetadata = (state: JobValues) => {
-        const { jobType, validationType, annotators_validators, pipeline, validators, categories } =
+        const { jobType, validationType, annotators_validators, pipeline, validators,annotators, categories } =
```
Comment (ammb95) [inline web/src/connectors/edit-job-connector/edit-job-connector.tsx:81]: Where is this annotators value being used?

File: web/src/connectors/edit-job-connector/edit-job-connector.tsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
     showNoExtractionTab
 }) => {
     const getMetadata = (state: JobValues) => {
```
Comment (ammb95) [inline web/src/connectors/edit-job-connector/edit-job-connector.tsx:80]: One suggestion of improvement [optional] would be to move this function to a separate file.

File: web/src/connectors/edit-job-connector/edit-job-connector.tsx (hunk 1 above)
Comment (Nosafi) [reply web/src/connectors/edit-job-connector/edit-job-connector.tsx:81]: I forgot to remove unnecessary variables, fixed.

File: web/src/connectors/edit-job-connector/edit-job-connector.tsx (hunk 2 above)
Comment (Nosafi) [reply web/src/connectors/edit-job-connector/edit-job-connector.tsx:80]: not sure if this is necessary here

**Review by ammb95 — APPROVED** [review]

**Pull request merged**